packages/components/src/tooltip/test/index.js

```diff
@@ -169,79 +164,71 @@ describe( 'Tooltip', () => {
+			const button = screen.getByRole( 'button' );
+			fireEvent.mouseOver( button );
+			const popoverBeforeTimeout =
+				container.getElementsByClassName( 'components-popover' );
 			expect( popoverBeforeTimeout ).toHaveLength( 0 );
 			expect( originalMouseEnter ).toHaveBeenCalledTimes( 1 );
 
 			// Popover does not yet exist after default delay, because custom delay is passed.
 			setTimeout( () => {
-				const popoverBetweenTimeout = wrapper.find( 'Popover' );
+				const popoverBetweenTimeout =
+					container.getElementsByClassName( 'components-popover' );
 				expect( popoverBetweenTimeout ).toHaveLength( 0 );
 			}, TOOLTIP_DELAY );
-
 			// Popover appears after custom delay.
 			setTimeout( () => {
-				const popoverAfterTimeout = wrapper.find( 'Popover' );
+				const popoverAfterTimeout =
+					container.getElementsByClassName( 'components-popover' );
 				expect( popoverAfterTimeout ).toHaveLength( 1 );
-
 				jest.runOnlyPendingTimers();
 				jest.useRealTimers();
 			}, 2000 );
 		} );
 
 		it( 'should show tooltip when an element is disabled', () => {
-			const wrapper = mount(
+			const { container } = render(
 				<Tooltip text="Show helpful text here">
 					<button disabled>Click me</button>
 				</Tooltip>
 			);
-			const button = wrapper.find( 'button[disabled]' );
-			const buttonNode = button.at( 0 ).getDOMNode();
-			const buttonRect = buttonNode.getBoundingClientRect();
-			const eventCatcher = wrapper.find( '.event-catcher' );
-			const eventCatcherNode = eventCatcher.at( 0 ).getDOMNode();
-			const eventCatcherRect = eventCatcherNode.getBoundingClientRect();
+
+			const button = screen.getByRole( 'button' );
+			const buttonRect = button.getBoundingClientRect();
+			const eventCatcher =
+				container.getElementsByClassName( 'event-catcher' )[ 0 ];
+			const eventCatcherRect = eventCatcher.getBoundingClientRect();
 			expect( buttonRect ).toEqual( eventCatcherRect );
 
-			eventCatcher.simulate( 'mouseenter', {
-				type: 'mouseenter',
-				currentTarget: {},
-			} );
+			fireEvent.mouseOver( eventCatcher );
 
 			setTimeout( () => {
-				const popover = wrapper.find( 'Popover' );
+				const popover =
+					container.getElementsByClassName( 'components-popover' );
 				expect( popover ).toHaveLength( 1 );
 			}, TOOLTIP_DELAY );
 		} );
 
 		it( 'should not emit events back to children when they are disabled', () => {
 			const handleClick = jest.fn();
-
-			const wrapper = mount(
+			const { container } = render(
 				<Tooltip text="Show helpful text here">
 					<button disabled onClick={ handleClick }>
 						Click me
 					</button>
 				</Tooltip>
 			);
 
-			const eventCatcher = wrapper.find( '.event-catcher' );
-			eventCatcher.simulate( 'click', {
-				type: 'click',
-				currentTarget: {},
-			} );
-
-			expect( handleClick ).toHaveBeenCalledTimes( 0 );
+			const eventCatcher =
+				container.getElementsByClassName( 'event-catcher' )[ 0 ];
+			fireEvent.click( eventCatcher );
```

Comment:
We should just try to fire a `click` event (through `@testing-library/user-event`) on the `button` element (as a user would behave like)

Reply:
Fixed by 19d029c 👍